daemons/controld/controld_te_callbacks.c

```diff
@@ -419,22 +419,22 @@ te_update_diff_v2(xmlNode *diff)
             crm_trace("Ignoring %s change for version field", op);
             continue;
 
-        } else if (strcmp(op, "move") == 0) {
+        } else if (pcmk__str_eq(op, "move", pcmk__str_none)) {
```

Comment:
FYI, this is perfectly fine, but using strcmp() is also fine when the items are guaranteed to be non-NULL

Reply:
That makes sense. I'll only keep the changes from `pcmk__str_casei` to `pcmk__str_none`.